src/libraries/Native/Windows/clrcompression/zlib/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.4.4)
+cmake_minimum_required(VERSION 3.14.5)
```

Comment:
This file is https://github.com/madler/zlib/blob/master/CMakeLists.txt . It is not used for building dotnet/runtime. Revert it.